easybuild/easyconfigs/f/FPM/FPM-1.15.1-GCCcore-12.2.0-Ruby-3.2.2.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'Bundle'
+
+name = 'FPM'
+version = '1.15.1'
+
+local_rubyver = '3.2.2'
+versionsuffix = '-Ruby-%s' % local_rubyver
```

Comment:
I think we can drop the `versionsuffix` as long as you stick to the default Ruby version for this toolchain (which is the case here).